planner/core/exhaust_physical_plans.go

```diff
@@ -2803,6 +2805,31 @@ func (la *LogicalAggregation) tryToGetMppHashAggs(prop *property.PhysicalPropert
 		}
 		hashAggs = append(hashAggs, agg)
 	}
+
+	// handle MPP Agg hints
+	var preferModes []AggMppRunMode
+	if la.aggHints.preferAggType&preferMPP1PhaseAgg > 0 {
+		preferModes = append(preferModes, Mpp1Phase)
+	} else if la.aggHints.preferAggType&preferMPP2PhaseAgg > 0 {
+		preferModes = append(preferModes, Mpp2Phase)
+	} else if la.aggHints.preferAggType&preferMPPTiDBAgg > 0 {
+		preferModes = append(preferModes, MppTiDB)
+	} else if la.aggHints.preferAggType&preferMPPScalarAgg > 0 {
+		preferModes = append(preferModes, MppScalar)
+	}
```

Comment:
If there are more one mpp aggregation hints. We can only use one?
Change this part to
```
if .. {}
if .. {}
if .. {}
if .. {}
```